.gitlab-ci.yml

```diff
@@ -8,13 +10,15 @@ stages:
 run-test:
   stage: test
   script:
-    - make test-all
-
+    - cp sample/vaultrc ~/.vaultrc
+    - chmod 700 ~/.vaultrc
```

Comment:
`chmod 700`? isn't it better to just set to `chmod 0600`?

Reply:
You're right. `chmod 0600` would be cleaner.